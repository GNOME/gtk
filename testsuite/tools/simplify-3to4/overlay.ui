<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk+" version="3.0"/>
  <object class="GtkOverlay">
    <property name="visible">1</property>
    <child type="overlay">
      <object class="GtkButton" id="button">
        <property name="visible">1</property>
        <property name="icon-name">document-new</property>
      </object>
      <packing>
        <property name="pass-through">True</property>
        <property name="index">0</property>
      </packing>
    </child>
    <child>
      <object class="GtkLabel" id="label"/>
    </child>
  </object>
</interface>
