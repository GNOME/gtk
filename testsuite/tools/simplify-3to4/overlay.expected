<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkOverlay">
    <child type="overlay">
      <object class="GtkButton" id="button">
        <property name="can-target">0</property>
        <property name="icon-name">document-new</property>
      </object>
    </child>
    <property name="child">
      <object class="GtkLabel" id="label">
        <property name="visible">0</property>
      </object>
    </property>
  </object>
</interface>
